fdbrpc/LoadBalance.actor.h

```diff
@@ -486,6 +486,12 @@ Future<REPLY_TYPE(Request)> loadBalance(
 				// server count is within "LOAD_BALANCE_MAX_BAD_OPTIONS". We
 				// do not need to consider any remote servers.
 				break;
+			} else if (badServers == alternatives->countBest() && i == badServers) {
+				TraceEvent("AllLocalAlternativesFailed")
+				    .suppressFor(1.0)
```

Comment:
Do we want to do 0.1 here?

Reply:
This can be quite noisy, as we observed in tests. For prod clusters that are larger, I worry too many such events even if we suppress with 0.1s. So I choose 1s, i.e., at most 3600 per server hour.

Reply:
Can you change it in main branch as well? Now we have diverging PR,  it is not a strictly cherry pick anymore

Reply:
Yes. Created #8007.